src/net/RTP/RTPPacket.cs

```diff
@@ -15,13 +15,18 @@
 //-----------------------------------------------------------------------------
 
 using System;
+#if !NETCOREAPP2_1_OR_GREATER || NETFRAMEWORK
+using System.Linq;
+#endif
 
 namespace SIPSorcery.Net
 {
     public class RTPPacket
     {
         public RTPHeader Header;
-        public byte[] Payload;

```

Comment:
The Payload property is on the public API so needs to stay available or will break a lot of existing app.

Reply:
Updated to make it public again